packages/libraries/core/src/utils/clipboard/write-on-clipboard.ts

```diff
@@ -0,0 +1,4 @@
+export function writeOnClipboard(textToWrite: string): Promise<void> {
+    console.log('test', textToWrite);
+    return navigator.clipboard.writeText(textToWrite);
```

Comment:
would be nice to wrap this in a `try catch` to prevent crash if the method get called from a non browser context

Reply:
ok but I think all the browser support the method https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText